docs/cli/edgedb_server_destroy.rst

```diff
@@ -0,0 +1,33 @@
+.. _ref_cli_edgedb_server_destroy:
+
+
+=====================
+edgedb server destroy
+=====================
+
+Stop an EdgeDB instance.
```

Comment:
It doesn't just stop. It removes all the data.

Reply:
Fixed